src/main/java/com/marklogic/kafka/connect/sink/BulkDataServicesSinkTask.java

```diff
@@ -0,0 +1,204 @@
+package com.marklogic.kafka.connect.sink;
```

Comment:
This is all the new stuff. I don't think anything else changed in terms of functionality, just had to move a lot of stuff around.